cfgov/unprocessed/js/organisms/Carousel.js

```diff
@@ -66,50 +67,42 @@ function Carousel( element ) {
     _updateDisplay( _currItemIndex, _currItemIndex + 1 );
   }
 
+
+  function _thumbnailClicked( event ) {
+    let node = event.target;
+    // Get thumbnail index
+    let i = 0;
+    while ( ( node = node.previousSibling ) !== null ) {
+      if ( node.nodeType === 1 ) {
+        ++i;
+      }
+    }
+    _updateDisplay( _currItemIndex, i );
```

Comment:
This should only be called if `_currItemIndex != i`, otherwise, if you click on the same thumbnail twice, it hides itself, leaving no item displayed:
![image](https://user-images.githubusercontent.com/654645/69280848-2f5f9200-0bb5-11ea-9793-002b3ad9a5af.png)

Reply:
Good catch, fixed!